crates/biome_cli/tests/cases/editorconfig.rs

```diff
@@ -164,3 +256,106 @@ indent_style = tab
         result,
     ));
 }
+
+#[test]
+fn should_have_cli_override_editorconfig() {
+    let mut fs = MemoryFileSystem::default();
+    let mut console = BufferConsole::default();
+
+    let editorconfig = Path::new(".editorconfig");
+    fs.insert(
+        editorconfig.into(),
+        r#"
+[*]
+max_line_length = 90
+"#,
+    );
+
+    let test_file = Path::new("test.js");
+    fs.insert(test_file.into(), r#"console.log("really long string that should break if the line width is <=90, but not at 100");"#);
```

Comment:
I think this change can reduce the error in `should_have_cli_override_editorconfig.snap`.

```suggestion
    fs.insert(test_file.into(), r#"console.log("really long string that should break if the line width is <=90, but not at 100");
"#);
```